index.js

```diff
@@ -1,34 +1,48 @@
 'use strict';
 
+const defer = require('p-defer');
+
+class AbortError extends Error {
+	constructor(message) { // eslint-disable-line no-useless-constructor
+		super(message);
```

Comment:
`this.name = 'AbortError';`

https://github.com/sindresorhus/eslint-plugin-unicorn/blob/master/docs/rules/custom-error-definition.md